tests/legacy-cli/e2e/tests/experimental/bep.ts

```diff
@@ -0,0 +1,10 @@
+import { ng } from '../../utils/process';
+import { expectFileToMatch } from '../../utils/fs';
+
+
+export default async function() {
+  await ng('build', '--build-event-log', 'bep-log.txt');
+
+  await expectFileToMatch('bep-log.txt', '{"id":{"started":{}},"started":{"command":"build",');
+  await expectFileToMatch('bep-log.txt', '{"id":{"finished":{}},"finished":{"finish_time_millis":');
```

Comment:
it would be nice to have a golden file test too, so there's a specimen of the BEP file that we create which is guaranteed up-to-date